src/sections/HomePortfolio/index.css

```diff
@@ -1,300 +1,284 @@
 @import "../../styles/all.css";
 
 .root {
+  width: 100%;
   @include Breakpoint-desktopOnly {
     margin-top: calc(calc($spacing-224 - $spacing-84) * -1);
   }
 }
 
-.project {
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-
-  margin-top: $spacing-224;
+.project-grid {
+  display: grid;
```

Comment:
As I've explained in our meeting, we preferentially use flex. To change order of columns with flex you can use order, check services section if you need.